examples/react-native-todo-list/App.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * @format
+ * @flow strict-local
+ */
+
+import React from "react";
+import {
+  SafeAreaView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  FlatList,
+  View,
+  KeyboardAvoidingView,
+  Platform,
+} from "react-native";
+
+import { useOthers, useList, useUpdateMyPresence } from "./liveblocks.config";
+
+import Header from "./src/components/Header";
+import Todo from "./src/components/Todo";
+import TextInputWithButton from "./src/components/TextInputWithButton";
+import NoMoreTodos from "./src/components/NoMoreTodos";
+
+const App = () => {
+  const todos = useList("todos");
+  const others = useOthers();
+  const updateMyPresence = useUpdateMyPresence();
+  const isSomeoneTyping = others
+    .toArray()
+    .some((user) => user.presence?.isTyping);
+
+  if (todos === null) {
+    return (
+      <View style={styles.noTodos}>
+        <Text>Loading...</Text>
+      </View>
+    );
+  }
+
+  return (
+    <SafeAreaView style={styles.backgroundContainer}>
+      <StatusBar barStyle="dark-content" />
+      <KeyboardAvoidingView
+        behavior={Platform.OS === "ios" ? "padding" : undefined}
+        style={styles.container}
+      >
+        <>
+          <Header whoIsHereCount={others?.count} />
+          {todos?.toArray().length > 0 ? (
```

Comment:
I missed it the first time but you can use `todos.length` instead of `todos.toArray().length`.

https://liveblocks.io/docs/api-reference/liveblocks-client#LiveList.length